dotCMS/src/main/java/com/dotcms/rest/api/v1/page/JsonMapper.java

```diff
@@ -0,0 +1,17 @@
+package com.dotcms.rest.api.v1.page;
+
+import com.dotmarketing.beans.Host;
+import com.dotmarketing.business.Permissionable;
+import com.dotmarketing.portlets.contentlet.model.Contentlet;
+import com.dotmarketing.portlets.htmlpageasset.model.HTMLPageAsset;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+public class JsonMapper {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1118)